src/main/java/net/visma/autopay/http/signature/SignatureContext.java

```diff
@@ -220,6 +249,48 @@ public Builder headers(Map<String, ?> headers) {
             return this;
         }
 
+        /**
+         * Adds HTTP headers.
+         * <p>
+         * Method offers compatibility with {@code HttpServletRequest} methods.
+         *
+         * @param headerNames   enumeration of HTTP servlet request header names
+         * @param headersGetter getter of HTTP servlet request headers enumeration
+         * @return This builder
+         */
+        public Builder headers(Enumeration<String> headerNames, Function<String, Enumeration<String>> headersGetter) {
```

Comment:
Could you add example call? Here, and in README.md: Full example -> Creating signature -> Signature context